DataToString.swift

```diff
@@ -129,10 +151,26 @@ func makeStringB(data: Data) -> String {
   } ?? ""
 }
 
+func dataToString(data: Data) -> String {
+    let holder = Holder(data: data)
+    
+    let string = String(_StringCore(
+        baseAddress: holder.pointer,
+        count: Int(data.count),
+        elementShift: 0,
+        hasCocoaBuffer: false,
+        owner: holder
+    ))
+    
+    return string
+}
+
+
 // Block to be scheduled
 func code(block: Int, loops: Int) -> () -> Void {
 return {
   var string: String?
+    var nsstring: NSString?
```

Comment:
can you remove this (unused) ref + the commented block?  I suspect method 0 is also not required.